test/presenters/notifications/setup/event.presenter.test.js

```diff
@@ -0,0 +1,179 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const RecipientsFixture = require('../../../fixtures/recipients.fixtures.js')
+
+// Thing under test
+const EventPresenter = require('../../../../app/presenters/notifications/setup/event.presenter.js')
+
+describe('Notifications Setup - Event presenter', () => {
+  let session
+  let recipients
+
+  beforeEach(async () => {
+    recipients = RecipientsFixture.recipients()
+
+    session = {
+      returnsPeriod: 'quarterFour',
+      removeLicences: [],
+      journey: 'invitations',
+      referenceCode: 'RINV-123',
+      recipients: [...Object.values(recipients)],
```

Comment:
changes will be added in another pr to provide this data to the session. 

The recipients (fetched and duplicates removed / contact_types marked as both etc) will be provided. 

And when a return cycle is selected we will determine the return cycle at one point and use that for the rest of the journey. We currently duplicate this logic in a few places.